CLAUDE.md

```
@@ -0,0 +1,8 @@
+<!--
+Claude Code는 AGENTS.md를 직접 읽지 않는다. 이 파일은 AGENTS.md를 Claude Code 컨텍스트로
+가져오기 위한 진입점일 뿐이며, 저장소 규칙은 AGENTS.md와 docs/에만 둔다.
+Windows에서는 symlink 생성에 관리자 권한이 필요하므로 @ 임포트 방식을 사용한다.
+이 주석은 컨텍스트에 주입되기 전에 제거되므로 토큰을 쓰지 않는다.
+-->
+
+@AGENTS.md
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)